README.md

```diff
@@ -1,10 +1,16 @@
 # Are we headless yet?
 
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+This is the source code to [AreWeHeadlessYet's website](https://areweheadlessyet.wagtail.org) frontend built upon [Next.js](https://nextjs.org/).
```

Comment:
Just changing the wording a bit for the SEO value ("wagtail headless" as link text).

```suggestion
This is the source code to [Wagtail’s Are we headless yet? website](https://areweheadlessyet.wagtail.org) frontend built with [Next.js](https://nextjs.org/).
```